protocol-designer/src/localization/en/alert.json

```
@@ -155,6 +155,10 @@
       "DROP_TIP_LOCATION_DOES_NOT_EXIST": {
         "title": "Attempting to drop tip in an unknown location",
         "body": "The Waste Chute or Trash Bin to drop tip in does not exist."
+      },
+      "MISSING_96_CHANNEL_TIPRACK_ADAPTER": {
+        "title": "Missing tiprack adapter for 96-channel",
+        "body": "A 96-channel cannot pick up a full tiprack without an adapter underneath."
```

@ecormany, thoughts on what this error text should say? It is triggered when you try to pipette from a tiprack with all 96 channels from a 96-channel pipette but the tiprack is missing the adapter underneath.

---

This mirrors what we've written in documentation:
```suggestion
      "MISSING_96_CHANNEL_TIPRACK_ADAPTER": {
        "title": "Missing 96-channel tip rack adapter",
        "body": "The tip rack must be placed in an adapter when picking up 96 tips simultaneously."
```

---

thank you!